README.md

```diff
@@ -1 +1,56 @@
-# demisto-sdk
\ No newline at end of file
+[![CircleCI](https://circleci.com/gh/demisto/demisto-sdk/tree/master.svg?style=svg)](https://circleci.com/gh/demisto/demisto-sdk/tree/master)
+
+# Demisto SDK 
+
+The Demisto SDK library can be used to manage your Demisto content with ease and efficiency.
+
+## Usage
+
+### Installation
+
```

Comment:
any prerequisites ?

Reply:
yet to be determined... except unify/extract, it's only working on our content repo for now